docs/specs/resolve.yml

```diff
@@ -348,3 +348,20 @@ outputs:
     { "content": "[!include[B1](~/b.md)]<p>Token</p>\n" }
   build.log: |
     ["warning","include-not-found","Cannot resolve '~/b.md' relative to 'docs/a.md'.","docs/a.md","",1,1]
+---
```

Comment:
nit: move this spec to `legacy.yml` to make it easier to find hacks like this?

Reply:
sure, we should get rid of it asap